config/manager/manager.yaml

```diff
@@ -62,6 +62,26 @@ spec:
       protocol: TCP
       port: 8888
       targetPort: 8888
+    - name: metrics
+      protocol: TCP
+      port: 8080
```

Comment:
@eloycoto is your work affected by this change ?

Reply:
No